github/GithubIntegration.py

```diff
@@ -35,8 +44,15 @@ def __init__(
         :param integration_id: int deprecated, use auth=github.Auth.AppAuth(...) instead
         :param private_key: string deprecated, use auth=github.Auth.AppAuth(...) instead
         :param base_url: string
+        :param timeout: integer
+        :param user_agent: string
+        :param per_page: int
+        :param verify: boolean or string
```

Comment:
Should this document what values of `verify` there are? I don't know why this would ever be a `str`, or what values it would have

Reply:
Ah, I tracked it down. this eventually flows to `requests.session` which takes the `verify` argument and uses it. I understand.